scipy/stats/tests/test_continuous.py

```diff
@@ -1113,7 +1113,7 @@ def test_rv_generic(self, i, distdata):
                 'johnsonsb', 'kappa4', 'ksone', 'kstwo', 'kstwobign', 'norminvgauss',
                 'powerlognorm', 'powernorm', 'recipinvgauss', 'studentized_range',
                 'vonmises_line', # continuous
-                'betanbinom', 'zipf', 'logser', 'skellam'}  # discrete
```

Comment:
I don't think these were ever slow, but they did fail when running all tests. Adding these distributions to the list prevented the tests from failing in regular CI, but to fix that we just needed to add a skip for `median` due to the left endpoint inversion issue.